src/main/java/dev/blaauwendraad/masker/json/ValueMaskerContext.java

```diff
@@ -58,4 +58,14 @@ public interface ValueMaskerContext {
      * Note: this INCLUDES the opening and closing quotes for string values 
      */
     String asString(int fromIndex, int length);
+
+    /**
+     * Create an {@link InvalidJsonException} with the given message and index relative to the value (i.e. an index
+     * between {@code 0} and {@link ValueMaskerContext#byteLength()}).
```

Comment:
Does this really make sense to give an index relative to the value in the message? Is it clear from a stacktrace which part is invalid exactly?

Reply:
It won't be that much clear, for example if a string value has an invalid sequence, then the stack trace will be at `maskString` and nothing more, so it won't be possible to know which string is that even